components/zerotier/sources/common/common.mjs

```diff
@@ -19,4 +19,44 @@ export default {
       ],
     },
   },
+  methods: {
+    _getNodeStatus(nodeId) {
+      return this.db.get(nodeId);
+    },
+    _setNodeStatus(nodeId, status) {
+      return this.db.set(nodeId, status);
+    },
+  },
+  async run({ $ }) {
+    const nodes = await this.zerotier.getNetworkNodes({
+      networkId: this.networkId,
+      $,
+    });
```

Comment:
The `$` variable is only used in actions. It'll just be `undefined` here, so no need to include it. Would you mind removing it from the sources `new-node-join` and `new-node-left` as well?
```suggestion
  async run() {
    const nodes = await this.zerotier.getNetworkNodes({
      networkId: this.networkId,
    });
```